web-components/src/components/input/Input.ts

```diff
@@ -469,7 +470,7 @@ export class Input extends FocusMixin(LitElement) {
   }
 
   messagesTemplate() {
-    return this.messages && !!this.messages.length
+    return !this.hideMessage && this.messages && !!this.messages.length
```

Comment:
@futurethang did we have this achieved differently?

Reply:
@yananym with TimePicker, I was able to just use css `::parts` to `display: none` on `md-input::part(message). But in this case, it doesn't seem possible as it is double nested. I tried md-menu-overlay md-input::part(message) but seems like parts doesn't work through two shadowDoms

Reply:
yeah you do it th same way Kevin did in Editable field. makes sense